roles/github/defaults/main.yml

```diff
@@ -131,6 +146,23 @@ github_run_infra_vm_service_deploy:
   arguments: "{{ github_kayobe_task_arguments }}"
   concurrency_group: infra
 
+github_octavia_certificates_schedule_trigger:
+  schedule:
+    cron: "{{ github_octavia_certificates_schedule }}"
+
+github_octavia_certificates_workflow_dispatch_trigger:
+  workflow_dispatch: "{{ github_octavia_certificates_input }}"
+
+github_run_kolla_ansible_octavia_certificates:
```

Comment:
This is not in the default list, so you'd need to maintain a copy of the default list in order to pull it in. That could easily get out of sync.

How about giving `github_workflows` the default/extra treatment to make it easier to extend?

Reply:
Done, will update docs to explain this